packages/contracts-bedrock/src/L1/L1ERC721Bridge.sol

```diff
@@ -18,14 +19,29 @@ contract L1ERC721Bridge is ERC721Bridge, ISemver {
     ///         by ID was deposited for a given L2 token.
     mapping(address => mapping(address => mapping(uint256 => bool))) public deposits;
 
+    /// @notice Address of the SuperchainConfig contract.
+    SuperchainConfig public superchainConfig;
+
     /// @notice Semantic version.
     /// @custom:semver 1.5.0
     string public constant version = "1.5.0";
 
     /// @notice Constructs the L1ERC721Bridge contract.
     /// @param _messenger   Address of the CrossDomainMessenger on this network.
-    /// @param _otherBridge Address of the ERC721 bridge on the other network.
-    constructor(address _messenger, address _otherBridge) ERC721Bridge(_messenger, _otherBridge) { }
+    constructor(address _messenger) ERC721Bridge(_messenger, Predeploys.L2_ERC721_BRIDGE) {
+        initialize(SuperchainConfig(address(0)));
```

Comment:
This is not true, it is supposed to take `address(0)` because the proxy pattern is being used and the sentinel contract should have null state set so people don't confuse it as the address they should interact with